client/src/entrypoints/admin/core.js

```diff
@@ -240,7 +240,7 @@ $(() => {
   });
 
   /* Functions that need to run/rerun when active tabs are changed */
-  $(document).on('shown.bs.tab', () => {
+  $(document).on('tab-changed', () => {
```

Comment:
This should probably change to non-jquery event listener. 

`document.addEventListener`

As it will avoid any issues with bubbling as jQuery events are not exactly handled the same as native DOM events.

Reply:
Thanks! I've made this change now